<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>DashBoard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>480</width>
     <height>560</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgb(122, 82, 208);

}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>560</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QFrame{
	background-color: rgb(255, 255, 255);
	border-radius: 10px;

}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QFrame{
	background-color: rgb(0, 200, 200);
	border-radius: 10px

}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <widget class="QPushButton" name="buttonPerfil">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>51</width>
            <height>51</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-image: url('../resource/usuario.png');
	background-repeat: none;
	background-position: center;
	background-color: rgb(0, 230, 230);
	border-radius: 25px;

}

QPushButton:hover{
	background-color: rgb(201, 201, 201);
}

QPushButton:pressed{
	background-color: rgb(243, 243, 243);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="labelUserDash">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>110</y>
            <width>441</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{
	color: rgb(255, 255, 255);

}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="buttonTrans">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>30</y>
            <width>71</width>
            <height>71</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	
	background-image: url('../resource/money.png');
	background-position: center;
	background-color:rgb(243, 243, 243);
	border-radius: 35px;
}

QPushButton:hover{
	background-image: url('../resource/money_hover.png');
}

QPushButton:pressed{
	background-image: url('../resource/money.png');
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonDeposita">
          <property name="geometry">
           <rect>
            <x>150</x>
            <y>30</y>
            <width>71</width>
            <height>71</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-image: url('../resource/deposito.png');
	background-position: center;
	background-color: rgb(243, 243, 243);
	border-radius: 35px;

}

QPushButton:hover{
	background-color: rgb(201, 201, 201);
}

QPushButton:pressed{
	background-color: rgb(243, 243, 243);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonSaca">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>30</y>
            <width>71</width>
            <height>71</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-image: url('../resource/saque.png');
	background-position: center;
	background-color: rgb(243, 243, 243);
	border-radius: 35px;

}

QPushButton:hover{
	background-color: rgb(201, 201, 201);
}

QPushButton:pressed{
	background-color: rgb(243, 243, 243);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QPushButton" name="buttonExtrato">
          <property name="geometry">
           <rect>
            <x>380</x>
            <y>32</y>
            <width>75</width>
            <height>71</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
	background-image: url('../resource/extrato.png');
	background-position: center;
	background-color: rgb(243, 243, 243);
	border-radius: 35px;

}

QPushButton:hover{
	background-color: rgb(201, 201, 201);
}

QPushButton:pressed{
	background-color: rgb(243, 243, 243);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>35</x>
            <y>110</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Transferir</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>110</y>
            <width>59</width>
            <height>15</height>
           </rect>
          </property>
          <property name="text">
           <string>Depositar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>110</y>
            <width>41</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Sacar</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>400</x>
            <y>110</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Extrato</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>10</y>
            <width>51</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>MS Shell Dlg 2</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Conta</string>
          </property>
         </widget>
         <widget class="QLabel" name="labelSaldo">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>141</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>R$ 0.33</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../src/resource/file.qrc"/>
 </resources>
 <connections/>
</ui>
